tests/lib/rules/prefer-query-by-disappearance.test.ts

```diff
@@ -0,0 +1,741 @@
+import rule, {
+  RULE_NAME,
+} from '../../../lib/rules/prefer-query-by-disappearance';
+import { createRuleTester } from '../test-utils';
+
+const ruleTester = createRuleTester();
+
+ruleTester.run(RULE_NAME, rule, {
+  valid: [
```

Comment:
You can also pass directly an element to `waitForElementToBeRemoved`, can you add a test for that? E.g.

```js
import { screen } from '@testing-library/react';

const button = screen.getByRole('button')
// ...
await waitForElementToBeRemoved(button)
```

Reply:
There is a test for that: https://github.com/testing-library/eslint-plugin-testing-library/pull/402/files#diff-85beea9fec5d5f9c3167a9bb1ca393a7a5c511cf3d4fec7acbe4a5558c1d1a79R97

Or do you mean specifically when having the element in a separate variable?

Reply:
And an equivalent invalid test: https://github.com/testing-library/eslint-plugin-testing-library/blob/71f541070b8f496ab1d6e6fc2f4ae1b184f6ab36/tests/lib/rules/prefer-query-by-disappearance.test.ts#L290

Reply:
I mean specifically having the element in a separate variable, so we cover how the rule behaves when a call expression is not found but an identifier.

Reply:
For clarification: the example I added doesn't have to be reported. If the rule finds an identifier as an argument of `waitForElementToBeRemoved` we can just assume is valid.